app/src/main/java/ru/myuniquenickname/myapplication/MainActivity.kt

```diff
@@ -3,16 +3,40 @@ package ru.myuniquenickname.myapplication
 import android.content.Intent
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.view.View
 import android.widget.TextView
+import ru.myuniquenickname.myapplication.fragments.FragmentMoviesDetails
+import ru.myuniquenickname.myapplication.fragments.FragmentMoviesList
+
+class MainActivity : AppCompatActivity() , FragmentMoviesList.TransactionsFragmentClicks{
+
+
+
+
 
-class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+         FragmentMoviesList().apply {
```

Comment:
May be better to split this construction into the fragment initialization and the transaction run?